src/test/regress/expected/with_optimizer.out

```diff
@@ -2554,6 +2554,51 @@ JOIN cte b USING (i);
  1
 (1 row)
 
+DROP TABLE with_test;
+-- Test shared modifying CTE behavior in utility mode
+CREATE TABLE with_test (i int, j int) DISTRIBUTED RANDOMLY;
+-- check when slice is not allocated (topslice is allocated only on QD)
+\! PGOPTIONS='-c gp_session_role=utility' psql -d regression -c 'EXPLAIN (SLICETABLE, COSTS OFF) WITH cte AS (INSERT INTO with_test VALUES (1, 2) RETURNING *) SELECT i FROM cte a JOIN cte b USING (i);'
```

Comment:
ORCA also should run on segment

Reply:
I've forgot about this output, thanks. Fixed.